plugins/out_stackdriver/stackdriver.c

```diff
@@ -344,6 +410,7 @@ static int stackdriver_format(const void *data, size_t bytes,
     char time_formatted[255];
     struct tm tm;
     struct flb_time tms;
+    flb_sds_t severity = flb_sds_create_size(10);
```

Comment:
do not allocate memory if you don't need it. If ctx->severity_key is NULL or not set, why to allocate and deallocate everytime ?, make the allocation conditional few lines below

Reply:
severity is now enum.